config/development.yml

```diff
@@ -0,0 +1 @@
+webAppHost: "WEB_APP_HOST"
```

Comment:
Set it to realistic value - say `http://localhost:8000` (please confirm with @vindru  on which port does frontend would run)